wgpu/src/backend/web.rs

```diff
@@ -340,6 +340,18 @@ fn map_primitive_state(primitive: &wgt::PrimitiveState) -> web_sys::GpuPrimitive
     //TODO:
     //mapped.unclipped_depth(primitive.unclipped_depth);
 
+    match primitive.polygon_mode {
+        wgt::PolygonMode::Fill => {}
```

Comment:
nitpick: I think `()` is generally preferable over `{}`; it's more direct.

Pushed as 48f701bb so you can autosquash it in, but if you disagree, feel free to drop it instead.

Reply:
It's not very often I see `()` over `{}` in this case. I personally prefer `{}` but don't care much as long as we consistently use one of them, the only reliable way of doing that being through rustfmt. However, there seems to be no such rule we could enable (https://rust-lang.github.io/rustfmt).

Reply:
I also prefer {}.  It's standard for "empty block" == do nothing, as opposed "returns the empty tuple" == do something by returning a nothing.